src/room.ts

```diff
@@ -10,7 +10,7 @@ import DailyIframe, {
   DailyRoomInfo,
   DailyEventObjectParticipantLeft,
 } from "@daily-co/daily-js";
-import { globalZoneID, standardTileSize } from "./config";
```

Comment:
Are we still using this `standardTileSize` somewhere else, or can we remove it from the **config** file as well ?

Reply:
It's still used in several models within the world